### PR TITLE
Added update.json for bootstrap-social

### PR DESCRIPTION
**Pull request by lipis**
Fixes https://github.com/jsdelivr/bot/issues/48

### CODE REVIEW

File: files/bootstrap-social/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "bootstrap-social",
+  "repo": "lipis/bootstrap-social",
+  "files": {
+    "include": ["./bootstrap-social.css"]
```
Comment (jimaek) [inline files/bootstrap-social/update.json]: just `bootstrap-social.css`

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Comment by lipis** [comment]
wow.. this @jsdelivrbot is quite cool dude.. he also squashing and merging stuff :dancer: :dancers: !!

**Pull request closed without merging**